esp/esp/program/modules/handlers/studentregphasezero.py

```diff
@@ -170,15 +167,9 @@ def studentlookup(self, request, tl, one, two, module, extra, prog, newclass = N
         if not 'username' in request.GET or 'username' in request.POST:
             return self.goToCore(tl)
 
-        return StudentRegPhaseZero.studentlookup_logic(request, tl, one, two, module, extra, prog, newclass)
-
-    @staticmethod
-    def studentlookup_logic(request, tl, one, two, module, extra, prog, newclass = None):
         limit = 10
-        from esp.web.views.json_utils import JsonResponse
 
-        q_phasezero = Q(phasezerorecord__program=prog)
-        queryset = ESPUser.objects.filter(q_phasezero).distinct()
+        queryset = ESPUser.objects.filter(Q(phasezerorecord__program=prog)).distinct()
```

Comment:
Sorry, should have been clearer -- you can just remove the `Q()` and do `filter(phasezerorecord__program=prog)`, it does the same thing.